matchboxdesigngroup/kindling
# Fix/move fonts outside dirs

## Changes proposed in this pull request

To streamline font management this PR moves the sample project fonts to the top level of `assets/font`. This matches the functionality of the Create Theme Block.

This PR moves the sample project fonts to `assets/fonts/` instead of storing the fonts in their own directories inside of `assets/fonts `assets/fonts/din-pro/`. The theme.json file is updated with the new file path.

Closes https://app.asana.com/0/1203895219649773/1206518703604971/f

## Pre-submit checklist

As the author of this pull request, I verify that:

- [x] I have set the target branch to `master`.
- [ ] I have verified that all automated tests pass or have provided a detailed comment about why I am submitting with a failed pipeline.

## Testing

### How to test the changes in this pull request

Follow the steps below to test the changes in this PR.

1. Pull this branch to a local site and verify the fonts are loading correctly.

### Functional tests

As the functional tester for this pull request, I verify that:

- [ ] The fonts should load and are moved to the top level of `assets/fonts`

Once testing is complete, notify the author of any failed tests and move the task to "Kick back" in Asana. If all tests pass, move the task to "Ready for Code Review" in Asana and tag a team member for code review.

### Code review

As the code reviewer for this pull request, I verify that:

- [ ] The code is free of obvious errors.
- [ ] The code is free of obvious duplication.

Once testing is complete, notify the author of any failed tests and move the task to "Kick back" in Asana or continue with the "merging" steps below.

## Merging

As the individual merging this pull request, I verify that:



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205909272470153

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1203895219649773/1206803619658538)
